utils/deployer/migrations/1_initial_migration.js

```diff
@@ -8,12 +8,18 @@ const DelegateFactory = artifacts.require('DelegateFactory');
 
 module.exports = async(deployer, network) => {
 
-  assert.notEqual(network, 'development', "Please choose a network besides 'development'")
+  let STAKING_TOKEN_ADDRESS
+  let WETH_ADDRESS 
 
-  network = network.toUpperCase()
-
-  let STAKING_TOKEN_ADDRESS = process.env[network + "_AST"]
-  let WETH_ADDRESS = process.env[network + "_WETH"]
+  if (network == 'development') {
+    STAKING_TOKEN_ADDRESS = "0xCC1CBD4f67cCeb7c001bD4aDF98451237a193Ff8"
```

Comment:
having hardcoded addresses doesn't make sense as these addresses will not exist for development.